docs/frequency-analysis/frequency-analysis.md

```diff
@@ -0,0 +1,10 @@
+Frequency analysis can be used to count letters, but in our case,
+we are counting the frequency of meaningful words and phrases and filtering out stopwords.
+It is a useful study within the field of computer science.
+This is due to its function because it is being capable of outputting
+the frequency of letters or groups of letters in a ciphertext.
+
+Frequency analysis is a study that can be found within our GatorMinor.
+A GatorMinor user can use frequency analysis by inputting the path_to_file
+for each markdown assignment he/she would like to run studies on.
+Then an user can run a a frequency analysis study on chose markdown assignments.
```

Comment:
Few things here:
1. `GatorMinor` should be `GatorMiner`
2. I think that the first statement `Frequency analysis is a study that can be found within our GatorMinor.` is starting to make things redundant, you can either revise it or remove it.
3. I think that `inputting` should be changed to something like `taking in`
4. `path_to_file` doesn't need to be in this format, you can simply use `directory path` or `files path`
Here is a suggestion:
```suggestion
GatorMiner uses frequency analysis by taking in the directory path
of a collection of markdown files. The user can then run
a frequency analysis study on the chosen markdown assignments.
```